### DISCUSSION TITLE
How to change/modify learning scheduler when use BaseFinetuning?

### DISCUSSION THREAD

**Discussion by ZeguanXiao** [Lightning-AI/pytorch-lightning#12993] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
When doing transfer learning, we not only want to change the learning rate between phases but also to reinitialize lr_scheduler. While [this comment ](https://github.com/PyTorchLightning/pytorch-lightning/issues/2006#issuecomment-793820085) gives an example to introduce a new lr_scheduler, trainer.new_schedule() seems not a valid API. And it is better to be compatible with the LearningRateMonitor callback.

Is any workaround to do this?
